doc/components/extrema.md

```diff
@@ -0,0 +1,73 @@
+# Extrema
+
+ROHD-HCL provides a component to find the extrema of a list of Logic.
+
+The component `Extrema` will determine the extremas (maximum or minimum) and their position in a list of [`Logic`]s. 
```

Comment:
In markdown files, you don't need to do the `[]` around things since it's not connected to `dart doc` linking capabilities.  Just wrapping with ticks "`" is enough. (remove the brackets)